NewForce-Cohort-5/tabloidfullstack-spring-of-deception
# cbg-CreatePost

# Description

There is a button to add a new post at the top of the page of Post.  When you click the button you're taken to a form that allows you to add a new post.  When you submit a new post you will be sent back to the post list.  The post you just created should be at the top of the list. Also changed the Home page to say hello to the full name of the current user logged in.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. Clone down the repository.
2. Run `npm install` while in the root directory
3. `git fetch -a`
4. `git checkout cbg-CreatePost`
5. Navigate to the root directory for this repo [tabloidfullstack-spring-of-deception](https://github.com/NewForce-Cohort-5/tabloidfullstack-spring-of-deception)
6. `start Tabloid.sln`
7. Make sure your db is created with seed data. Run [01_Tabloid_Create_DB.sql](https://github.com/NewForce-Cohort-5/tabloidfullstack-spring-of-deception/blob/main/SQL/01_Tabloid_Create_DB.sql) and [02_Tabloid_Seed_Data.sql](https://github.com/NewForce-Cohort-5/tabloidfullstack-spring-of-deception/blob/main/SQL/02_Tabloid_Seed_Data.sql) queries in Visual Studio.
8. Start the server.
9. Navigate to [client](https://github.com/NewForce-Cohort-5/tabloidfullstack-spring-of-deception/tree/main/Tabloid/client)
10. `npm start`
11. Log into any account (foo@bar.com is fine)  You can leave the password field blank.
12. Click Posts at the top of the page in the navbar
13. Click the 'Create a new post' button at the top of the page.
14. Fill all the fields out and click submit.
15. You will be redirected to the Posts list.  The post you just created should be at the very top.



# Checklist:
